standard.md

```diff
@@ -4,19 +4,19 @@
+Indicator | Requirement
+--- | ---
+**1. Relevance to Sustainable Development Goals** | All projects must indicate the [Sustainable Development Goals](https://sdgs.un.org/goals) (SDGs) they are relevant to, and provide supporting links/documentation to support their relevance.
+**2. Use of Approved Open Licenses** | Projects must demonstrate the use of an approved open license. For open source software, only [OSI approved licenses](https://opensource.org/licenses) are accepted. For open content the use of a [Creative Commons license](https://creativecommons.org/licenses/) is required. While we encourage projects to use a license that allows for both derivatives and commercial reuse ([CC-BY](https://creativecommons.org/licenses/by/4.0/) and [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)), or dedicate content to the public domain ([CC0](https://creativecommons.org/choose/zero/)); licenses that do not allow for commercial reuse ([CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/)) are also accepted. For open data, an [Open Data Commons approved license](https://opendefinition.org/licenses/) is required. [The full license list is here for reference](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md).
+**3. Clear Ownership** | Ownership of everything the project produces must be clearly defined and documented. For example, through copyright, trademark or other publicly available information.
+**4. Platform Independence** | If the project has mandatory dependencies that create more restrictions than the original license, the project(s) must be able to demonstrate independence from the closed component(s) and/or indicate the existence of functional, open alternatives.
+**5. Documentation** | The project must have documentation of the source code, use cases, and/or functional requirements. For content, this should include all relevant/compatible apps, software, or hardware required to access the content, and instructions regarding how to use it. For software projects, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how data was collected, and how it should be interpreted.
+**6. Mechanism for Extracting Data** | If the project has non personally identifiable information (PII) there must be a mechanism for extracting or importing non PII-data from the system in a non-proprietary format.
```

Comment:
the hyphen in `non PII-data` seems off-placed. I believe it should read `non-PII data`

Reply:
Thank you!